docs/c_cpp_properties.json

```
@@ -0,0 +1,113 @@
+{
+    "configurations": [
+        {
+            "name": "N64 oot-gc-us",
```

N64 GC? :thonk:

---

"N64" as in "cross compiling" or "Not Windows/Linux x86 PC"
any suggestion?

---

I would drop the N64 part entirely
```suggestion
            "name": "oot-gc-us",
```

---

fwiw I agree-- I think its fair that the "building for n64" part is implied with this project. For anyone using decomp for anything else, they can change the names